### DISCUSSION TITLE
(Closed) How to carry server_config.yaml into effect after being modified?

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15699] [Q&A and General discussion]
Hi, I just modified server_config.yaml file. Should I remove my milvus docker container and deploy it again, or just docker restart my milvus container after I modified server_config.yaml?

**Comment by yhmo** [comment]
Two ways:
1. Firstly go in to cmd of the milvus container: ` docker exec -it [container id] bash`
    Secondly, modify the server_config.yaml under the /var/lib/milvus/conf/server_config.yaml
    Third, restart the container, the modification will take effect: `docker restart [container id]`
    
2. Firstly, mapping the /var/lib/milvus/conf to an external path:
`docker run -it -p 19530:19530 -v /home/xxx/milvus/conf:/var/lib/milvus/conf  milvusdb/milvus:1.1.1-cpu-d061621-330cc6`
    Secondly, modify the /home/xxx/milvus/conf/server_config.yaml
    Third, restart the container, the modification will take effect: `docker restart [container id]`

**Reply by xiaojuan8** [reply]
👌got it, so nice!
